packages/angular_devkit/core/node/host.ts

```diff
@@ -264,25 +264,28 @@ export class NodeJsSyncHost implements virtualFs.Host<fs.Stats> {
   delete(path: Path): Observable<void> {
     return this.isDirectory(path).pipe(
       concatMap(isDir => {
-        if (isDir) {
-          // Since this is synchronous, we can recurse and safely ignore the result.
-          for (const name of fs.readdirSync(getSystemPath(path))) {
-            this.delete(join(path, name)).subscribe();
-          }
-          try {
-            fs.rmdirSync(getSystemPath(path));
-          } catch (error) {
-            return throwError(error);
-          }
-        } else {
-          try {
+        // TODO: remove this try+catch when issue https://github.com/ReactiveX/rxjs/issues/3740 is
+        // fixed.
+        try {
+          if (isDir) {
+            const dirPaths = fs.readdirSync(getSystemPath(path));
+            const rmDirComplete = new Observable((obs) => {
+              fs.rmdirSync(getSystemPath(path));
```

Comment:
The try/catch needs to be around this call inside the observable constructor.
```
try {
  fs.rmdirSync(getSystemPath(path));
  obs.complete();
} catch (e) {
  obs.error(e);
}
```

Reply:
Done